modules/tensor_mechanics/include/materials/ComputeMultipleInelasticStress.h

```diff
@@ -175,5 +176,12 @@ class ComputeMultipleInelasticStress : public ComputeFiniteStrainElasticStress
 
   /// is the elasticity tensor guaranteed to be isotropic?
   bool _is_elasticity_tensor_guaranteed_isotropic;
-};
 
+  /// whether to include damage index or not
+  bool _include_damage;
```

Comment:
Couldn't you remove this parameter and just use `if (_damage_model)` as a check?